system/Cache/CacheInterface.php

```diff
@@ -83,7 +83,7 @@ public function clean();
      * The information returned and the structure of the data
      * varies depending on the handler.
      *
-     * @return mixed
+     * @return array|bool|object|null
```

Comment:
Do we need `true`? 
I mean not `bool` but `false`.

Reply:
Ok, I got it and I have been change `bool` to `false`.
I appreciated it!

Reply:
Thank you for quick fix!